source/rst/lss_examples.rst

```diff
@@ -4,33 +4,33 @@ Methods of the Linear State Space Class
 
 QuantEcon contains a class which has various methods for operating on
 Linear State Space Models (such models are explained in detail `in this
-lecture <https://lectures.quantecon.org/py/linear_models.html>`__).
+lecture <https://lectures.quantecon.org/py/linear_models.html>`__)
 
 In this notebook, we will illustrate the methods that can be used on
-instances of this class.
+instances of this class
 
 To illustrate these methods, we will use Paul Samuleson's (1939)
-multiplier-accelerator model.
+multiplier-accelerator model
 
 Samuelson's (1939) multiplier-accelerator model
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
This should be a subheading with `---`. You will also need to capitalise.

Reply:
@natashawatkins Is it fine if we do capitals tomorrow with the code?

Reply:
@natashawatkins are you referring to the commas in arguments or anywhere else?